virttest/shared/cfg/machines.cfg

```diff
@@ -127,6 +127,6 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
-        # The main disc is usually /dev/dasda device
-        indirect_image_blacklist = "/dev/dasda[\d]*"
+        # The main disc is usually /dev/sda device
```

Comment:
The comment should also be updated, other LGTM.

Reply:
Thanks for highlight, updated